types/svg2ttf/svg2ttf-tests.ts

```diff
@@ -0,0 +1,20 @@
+const dummySvgFontString = `
```

Comment:
```suggestion
import  svg2ttf = require('svg2ttf');

const dummySvgFontString = `
```
this is how it should be used and tested, see README sample:
https://github.com/fontello/svg2ttf#svg2ttfsvgfontstring-options---buf
```ts
/// <reference types="node" />
import  fs = require('fs');
import svg2ttf = require('svg2ttf');

const ttf = svg2ttf(fs.readFileSync('myfont.svg', 'utf8'), {});
fs.writeFileSync('myfont.ttf', new Buffer(ttf.buffer));
```